validate/action.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Validate metadata'
+description: 'Validates tagged package version is a valid semver value'

```

Comment:
Not sure how useful this check is. Not all our packages follows strict semver convention.

Reply:
A requirement of this change was to maintain the [existing validation logic](https://github.com/spacetelescope/action-publish_to_pypi/blob/master/validate_version.py). If it's not necessary and can be removed, I'll be more than happy to remove it. Less code, less bugs!

Reply:
A requirement of this change was to maintain the [existing validation logic](https://github.com/spacetelescope/action-publish_to_pypi/blob/master/validate_version.py). If it's not necessary and can be removed, I'll be more than happy to remove it. Less code, less bugs!

Reply:
For now I'll leave this in. We can make it a no-op or opt-in later if we so choose.